src/spyglass/lfp/lfp_merge.py

```diff
@@ -0,0 +1,53 @@
+import datajoint as dj
+from spyglass.lfp.v1 import LFP, ImportedLFP
+
+schema = dj.schema("lfp")
+
+
+@schema
+class LFPOutput(dj.Manual):
+    definition = """
+    lfp_id: uuid
+    ---
+    stream: varchar(40)
```

Comment:
Just out of curiosity what is stream?

Reply:
@khl02007 Actually I didn't understand this when you put it in before as a suggestion as well. Can you comment on this?

Reply:
In theory, I can see this being the name of the upstream table so it know which part table to look in. For instance, stream = "LFPv1" or stream = "ImportedLFP". But would be curious what your thinking is @khl02007 ?